docs/manuals/core/config/global-config.md

```diff
@@ -52,4 +54,56 @@ In this example, we set custom values for the _root_folder_, _storage_folder_, _
 _read_entity_retry_ parameters. Note that most of the time the default values can be used. In lines 7-8, two custom
 properties are specified: a _version_name_ and an _application_name_.
 
+=== "Repository configuration"
```

Comment:
I'm not sure if this is the best place or if we should add a separated section just for repositories. Since repositories are not documented in the reference manual, because they are protected classes of taipy, this documentation felt like it doesn't have a proper place.

Let me know what you think and what else should we add.

Reply:
I propose to have a dedicated section. I just pushed a proposal on your branch directly. 
I also updated the layout.
Can you review it?
Thx

Reply:
I believe a dedicated section would be better. I just pushed some changes. Please review them. Thx